api/tests/tooling/domain-builder/factory/build-competence-airtable-data-object.js

```diff
@@ -1,9 +1,13 @@
 module.exports = function buildCompetenceAirtableDataObject({
   id = 'recsvLz0W2ShyfD63',
   name = 'Mener une recherche et une veille d’information',
+  nameFrFr = 'Mener une recherche et une veille d’information',
+  nameEnUs = 'Mener une recherche et une veille d’information',
   index = '1.1',
   areaId = 'recvoGdo7z2z7pXWa',
   description = 'Some description',
+  descriptionFrFr = 'Some description',
+  descriptionEnUs = 'Some description',
```

Comment:
```suggestion
  nameFrFr = 'Mener une recherche et une veille d’information',
  nameEnUs = 'Browsing, searching and filtering data, information and digital content',
  index = '1.1',
  areaId = 'recvoGdo7z2z7pXWa',
  description = 'Une description',
  descriptionFrFr = 'Une description',
  descriptionEnUs = 'Some description',
```

Je propose de traduire dans le builder qu'il n'y ait pas de problème d'incompréhension plus tard